docs/guides/what-is-certifaction.md

```diff
@@ -12,11 +12,11 @@ Just want to try it out? Skip to the [Quickstart](./getting-started).
 
 [Certifaction](https://certifaction.com) is the Privacy-first electronic signature with
 
-- end to end encryption of documents,
+- end-to-end encryption of documents,
 - legally valid identity check,
-- Digital Twin bridging printed document with their digital original.
+- Digital Twin connecting printed documents with their digital original.
```

Comment:
I would keep the "bridging" here because of "creating bridge" is a better image, IMHO.